backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -780,6 +781,7 @@ public async System.Threading.Tasks.Task<Project> SetStatusAsync(Project project
             // Update a project
             originalProject.StatusId = toStatus.Id;
             originalProject.Status = toStatus;
+            originalProject.Metadata = project.Metadata;
```

Comment:
Something to think about: From what I could tell, it shouldn't be necessary to populate originalProject with metadata values either in the update or setStatus functions as those values don't seem to be required for business logic. Correct me if I'm wrong.

Reply:
I'd think you'd need it in `Add` and `Update`, otherwise you'll never have values there.  Since `SetStatus` also updates, you'll probably need it there too.

There are some possible business rules related to required date values before proceeding between states, but we may only enforce them currently on the frontend.  Ideally we'd enforce in the backed too.